client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -48,48 +48,47 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
           <TextField
             id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
+            label={<Typography className={classes.label}>EMAIL ADDRESS</Typography>}
```

Comment:
Please avoid using CAPITALCASE text as some screen readers interpret them as abbreviations. You can use a CSS class to transform the text case instead :)

Ditto throughout!